File: ArchitectureComponents/Room/CRUDWithRoom/app/src/main/java/com/dodemy/android/crudwithroom/MainActivity.java
package com.dodemy.android.crudwithroom;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        List<Item> items = new ArrayList<>();

        FloatingActionButton fab = findViewById(R.id.fab);
        RecyclerView recyclerView = findViewById(R.id.recycler_view);
        ItemAdapter adapter = new ItemAdapter(this);
        recyclerView.setAdapter(adapter);
        recyclerView.setLayoutManager(new LinearLayoutManager(this));

//        ItemRoomDatabase db = ItemRoomDatabase.getDatabase(MainActivity.this);
//        ItemDao dao = db.itemDao();
//        items = dao.getAllItems();
//        adapter.setItems(items);

        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, AddItemActivity.class));
            }
        });
    }
}

File: WebViewInKotlin/app/src/main/java/com/dodemy/android/webviewinkotlin/MainActivity.kt
package com.dodemy.android.webviewinkotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val pageUrl = "https://google.com/"
        launchWebView.setOnClickListener {
            val intent = WebActivity.newIntent(this, pageUrl)
            startActivity(intent)
        }
    }

}
